plugin/php/customizer/class-controls.php

```diff
@@ -133,12 +149,14 @@ public function add_panel() {
 	 */
 	public function add_sections() {
 		$sections = [
-			'style'          => __( 'Starter Styles', 'material-design' ),
-			'style_settings' => __( 'Style', 'material-design' ),
-			'colors'         => __( 'Color Palette ', 'material-design' ),
-			'typography'     => __( 'Typography (Font Styles)', 'material-design' ),
-			'corner_styles'  => __( 'Shape Size', 'material-design' ),
-			'icons'          => __( 'Icon Styles', 'material-design' ),
+			'style'           => __( 'Starter Styles', 'material-design' ),
+			'style_settings'  => __( 'Style', 'material-design' ),
+			'colors'          => __( 'Color Palette', 'material-design' ),
+			'typography'      => __( 'Typography (Font Styles)', 'material-design' ),
+			'corner_styles'   => __( 'Shape Size', 'material-design' ),
+			'icons'           => __( 'Icon Styles', 'material-design' ),
+			'dark_colors'     => __( 'Dark Color Palette', 'material-design' ),
+			'contrast_colors' => __( 'High Contrast Color Palette', 'material-design' ),
```

Comment:
There are some high contrast reference in PR. though if it's certainly coming later than we can keep it.